modules/storage-page-memory/src/main/java/org/apache/ignite/internal/storage/pagememory/PersistentPageMemoryStorageEngine.java

```diff
@@ -196,6 +197,11 @@ public PersistentPageMemoryTableStorage createMvTable(TableConfiguration tableCf
         return new PersistentPageMemoryTableStorage(this, tableCfg, regions.get(dataStorageView.dataRegion()));
     }
 
+    /** {@inheritDoc} */
+    @Override public TxnStateTableStorage createTxnStateTableStorage(TableConfiguration tableCfg) {
+        throw new UnsupportedOperationException("Transaction state storage can be created only using RocksDb storage engine.");
```

Comment:
Now this is one huge abstraction leak. Page memory storage shouldn't know that RocksDB even exist